packages/server/src/deprecated/interop.ts

```diff
@@ -145,28 +150,31 @@ export type MigrateRouter<
   >
 >;
 
-export type MigrateOldRouter<TRouter extends AnyOldRouter> =
-  TRouter extends OldRouter<
-    infer TInputContext,
-    infer TContext,
-    infer TMeta,
-    infer TQueries,
-    infer TMutations,
-    infer TSubscriptions,
-    infer TErrorShape,
-    infer Transformer
-  >
-    ? MigrateRouter<
-        TInputContext,
-        TContext,
-        TMeta,
-        TQueries,
-        TMutations,
-        TSubscriptions,
-        TErrorShape,
-        Transformer
-      >
-    : never;
+export type MigrateOldRouter<
+  TRouter extends AnyOldRouter,
+  TNamespaceDelimiter extends string = '.',
```

Comment:
This is all I added in this block, prettier has a different mood nowadays I guess.